TACC-Cloud/geoapi
# task/WG-196: add react-based hazmapper service

## Overview: ##

This PR 
* adds react-based hazmapper service as part of [WG-196](https://tacc-main.atlassian.net/browse/WG-196) and https://github.com/TACC-Cloud/hazmapper/pull/189.
* also adds automatic deployment of hazmapper/taggit/geoapi on Dev using [watchtower](https://github.com/containrrr/watchtower) for  [WG-163](https://tacc-main.atlassian.net/browse/WG-163)


 In https://github.com/TACC-Cloud/hazmapper/pull/189, a new image for react-based hazmapper is added.  This serves that service alongside hazmapper and taggit. Our VM `hazmapper.tacc.utexas.edu` has been updated to have:

- dev-react point to dev.geoapi-services:82
- staging-react point to staging.geoapi-services:82
- hazmapper-react point to prod.geoapi-services:82

## Related Jira tickets: ##

* [WG-196](https://tacc-main.atlassian.net/browse/WG-196)
* [WG-163](https://tacc-main.atlassian.net/browse/WG-163)

## Summary of Changes: ##

## Testing Steps: ##
1. Check that https://hazmapper.tacc.utexas.edu/dev-react is working as expected.
2. To see [watchtower](https://github.com/containrrr/watchtower) working you can see the logs of that container on portal@dev.geoapi-services.tacc.utexas.edu.  You can see it checking every 300s and also it updating containers if it finds something new.
